### DISCUSSION TITLE
Using pan down to close with multiple BottomSheetModals

### DISCUSSION THREAD

**Discussion by willcray** [gorhom/react-native-bottom-sheet#782] [Q&A]
In the `MapExample`, there are two modals stacked on top of each other. That are fluidly presented or dismissed via interacting with the buttons in the modals.

How could you extend this example to make it so that if you pan down on either modal, then it dismisses both modals and returns to the previous screen (i.e. whatever is sitting underneath the modals)? In `MapExample`, this would be a full-screen version of the map.

**Comment by hirbod** [comment]
You could setup `onDismiss={bottomSheetDismissed}` on your desired BottomSheet and trigger dismissAll(). This should be fast enough to prevent the underlying BottomSheet from poppin up
